src/common/schemas/output/ledger-closed.json

```diff
@@ -4,15 +4,48 @@
   "description": "A ledgerClosed event message",
   "type": "object",
   "properties": {
-    "feeBase": {"type": "integer", "minimum": 0},
-    "feeReference": {"type": "integer", "minimum": 0},
-    "ledgerHash": {"$ref": "hash256"},
-    "ledgerVersion": {"$ref": "ledgerVersion"},
-    "ledgerTimestamp": {"type": "string", "format": "date-time"},
-    "reserveBase": {"type": "integer", "minimum": 0},
-    "reserveIncrement": {"type": "integer", "minimum": 0},
-    "transactionCount": {"type": "integer", "minimum": 0},
-    "validatedLedgerVersions": {"type": "string"}
+    "feeBase": {
+      "type": "integer",
+      "minimum": 0,
+      "description": "Base fee, in drops."
```

Comment:
I have to fix this to make it return XRP, but since it's a breaking change, I was going to leave it for a separate commit.